hw/ip/edn/dv/edn_sim.core

```diff
@@ -0,0 +1,26 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:edn_sim:0.1"
+description: "EDN DV sim target"
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:ip:edn
+
+  files_dv:
+    depend:
+      - lowrisc:dv:edn_test
+      - lowrisc:dv:edn_sva
+    files:
+      - tb.sv
+    file_type: systemVerilogSource
+
+targets:
+  sim:
+    toplevel: tb
+    filesets:
+      - files_rtl
+      - files_dv
+    default_tool: vcs
```

Comment:
@cindychip is updating the template of this file, but this is created before that one is in.
can you add the update from this [PR](https://github.com/lowRISC/opentitan/pull/4023/files) and also add the IP to hw/top_earlgrey/lint/top_earlgrey_dv_lint_cfgs.hjson?

Reply:
Done